src/detect-engine-content-inspection.c

```diff
@@ -247,6 +247,10 @@ int DetectEngineContentInspection(DetectEngineCtx *de_ctx, DetectEngineThreadCtx
                 prev_buffer_offset = 0;
             }
 
+            if (cd->flags & (DETECT_CONTENT_DISTANCE_BE|DETECT_CONTENT_OFFSET_BE| DETECT_CONTENT_DEPTH_BE)) {
```

Comment:
can you add a comment on the why

also wondering why this includes distance but the mpm patch doesn't?

style: space in flags

Reply:
Will address.